x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/pages/rule_management/index.tsx

```diff
@@ -117,11 +127,18 @@ const RulesPageComponent: React.FC = () => {
                 <AddElasticRulesButton isDisabled={!canUserCRUD || loading} />
               </EuiFlexItem>
               <EuiFlexItem grow={false}>
-                <EuiToolTip position="top" content={i18n.UPLOAD_VALUE_LISTS_TOOLTIP}>
+                <EuiToolTip
+                  position="top"
+                  content={
+                    cantCreateNonExistentListIndex
+                      ? i18n.UPLOAD_VALUE_LISTS_PRIVILEGES_TOOLTIP
+                      : i18n.UPLOAD_VALUE_LISTS_TOOLTIP
```

Comment:
Nit: Ideally, the tooltip should also take into account `!canWriteListsIndex` and `!canUserCRUD` and show appropriate messages in these cases, because we disable the button based on `isImportValueListDisabled` and not only `cantCreateNonExistentListIndex`.

Reply:
Haven't touched this part, since there is already missing privileges banner, that looks like cover these cases

![image](https://github.com/elastic/kibana/assets/92328789/ac68f0f6-99f5-486b-a453-caa004cbd44b)

missing lists index privileges and `canUserCRUD` I think is detection privileges. So, basically, tooltip would duplicate banner content. 
But banner does not cover case, when index(DS) doesn't exist and user can't create it.
So, for now I tend to leave message same as before and just cover missing part.